### ISSUE TITLE
Web Platform Installer does not re-install Runtime 5.4.145.9494 after uninstalling once.

### ISSUE THREAD

**Issue by ob1jedi** [Azure/service-fabric-issues#147]
I installed Service Fabric SDK 2.4.145, which installed the runtime aswell. Wanting to test the standalone package I uninstalled the SDK, since the msi version conflicts with the standalone version. After I had tested the standalone version my Visual Studio started complaining that I didn't have SF installed, so I attempted to re-install the SDK via the msi using the same method I had before. 
Upon re-install, WebPI refused to install the SDK saying that it needed Runtime 5.4.145.9494. I couldn't get the Runtime package via WebPI and any official downloads only provided an exe which started up Web PI and again didn't download the Runtime, and proceeded to complain that it needed the Runtime. (There was also an authorization error at some point during WebPI's installation, possibly caused by company web-sensing, but then the error changed again to requiring-the-runtime)

Bottom Line: Please provide access to an independent package of the Runtime (not the SDK and Visual Studio tools, and pretty please, not through WebPI)

Thanks
Kind Regards

**Comment by MikkelHegn** [comment]
Check the detailed release notes which are linked to from this blog post. These contains links to msi installer packages. https://blogs.msdn.microsoft.com/azureservicefabric/2016/12/15/release-of-sdk-2-4-145-and-runtime-5-4-145-for-windows/

**Comment by ob1jedi** [comment]
Hi Mikkel, thanks for your response.
The WebPI installer is what I was having a problem with, and the link you provided points to a package that automatically starts the WebPI. 
I've since figured out what the problem was though. 
I was getting an "unauthorized" message when WebPI was attempting to download the C++ distributable (this may have been caused by company web-sensing), and as a result was preventing WebPI from getting the rest of the package (the Runtime). So to remedy the problem, I opened up Fiddler while running the WebPI and got the address of the C++ package from Microsoft. Pasting the link into my browser, I was able to download the C++ package and install it. After installing the C++ package, I ran the SF-install-exe again, and WebPI successfully completed its installation of Service Framework.

Thanks :)
Kind Regards

**Comment by MikkelHegn** [comment]
Great you got it to work. Regarding the link (common thing) There is a link in the blog post to a detailed release notes (a Word document). In that document there is a link to the download the MSI packages. It' two hubs away :-)
